test/stripe/api_resource_test.rb

```diff
@@ -271,14 +271,6 @@ class NestedTestAPIResource < APIResource
         assert_equal c.created, 12_345
       end
 
-      should "accessing a property other than id or parent on an unfetched object should fetch it" do
-        stub_request(:get, "#{Stripe.api_base}/v1/charges")
-          .with(query: { customer: "cus_123" })
-          .to_return(body: JSON.generate(customer_fixture))
-        c = Stripe::Customer.new("cus_123")
-        c.charges
-      end
```

Comment:
I've removed this test because I don't think it does what it says it does. `charges` is (was) a method that fires an API request, not a field that is automatically fetched when the object is unfetched as the description implies.